prime-dotnet-webapi/ViewModels/Enrollee/_EnrolleeMappingProfile.cs

```diff
@@ -23,10 +24,14 @@ public EnrolleeMappingProfile()
                     !src.Submissions.OrderByDescending(s => s.CreatedDate).FirstOrDefault().Confirmed
                     && src.PreviousStatus.StatusCode == (int)StatusType.RequiresToa
                 ))
+                .ForMember(dest => dest.LinkedEnrolleeId, opt => opt.MapFrom(src => (src.PaperEnrolment == null) ? src.LinkedEnrolment.EnrolleeId : src.PaperEnrolment.PaperEnrolleeId))
+                .ForMember(dest => dest.PossiblePaperEnrolmentMatch, opt => opt.MapFrom(src => unlinkedPaperEnrolments.Any(e => e.DateOfBirth.Date == src.DateOfBirth.Date)))
                 .ForMember(dest => dest.Confirmed, opt => opt.MapFrom(src => src.Submissions.OrderByDescending(s => s.CreatedDate).FirstOrDefault().Confirmed == true));
 
             CreateMap<Enrollee, EnrolleeDTO>()
                 .ForMember(dest => dest.Confirmed, opt => opt.MapFrom(src => src.Submissions.OrderByDescending(s => s.CreatedDate).FirstOrDefault().Confirmed == true))
+                .ForMember(dest => dest.LinkedEnrolleeId, opt => opt.MapFrom(src => (src.PaperEnrolment == null) ? src.LinkedEnrolment.EnrolleeId : src.PaperEnrolment.PaperEnrolleeId))
+                .ForMember(dest => dest.PossiblePaperEnrolmentMatch, opt => opt.MapFrom(src => (src.GPID != null && src.GPID.Contains("NOBCSC")) ? false : unlinkedPaperEnrolments.Any(e => e.DateOfBirth.Date == src.DateOfBirth.Date)))
                 .ForMember(dest => dest.HasNewestAgreement, opt => opt.MapFrom(src => newestAgreementIds.Any(id => id == src.CurrentAgreementId)));
 
             CreateMap<EnrolleeDTO, EnrolleeViewModel>();
```

Comment:
I am getting "Multiple possible paper enrolment matches" warning when there's only one paper enrolment with matching birthdate, but no matching GPID

**Enrollee table:**
![image](https://user-images.githubusercontent.com/34261366/143137866-771e6332-4654-48f0-a0ae-13266a579320.png)

**EnrolleeLinkedEnrolment table:**
![image](https://user-images.githubusercontent.com/34261366/143137953-38d27049-b506-4df1-8ed5-1fed75448102.png)

**Warning:**
![image](https://user-images.githubusercontent.com/34261366/143138030-c0996314-c1ed-4342-8f35-ce6f834980e7.png)

Maybe this can be solved by wording it differently if one potential match is found. There's another ticket that I'm working on where we display a link to the potential match only, but not to the confirmed links

Reply:
Agree. Maybe the wording should be ` Possible paper enrolment matches`